cpp/src/parquet/metadata.h

```diff
@@ -182,6 +184,28 @@ class PARQUET_EXPORT ColumnChunkMetaData {
   std::unique_ptr<ColumnChunkMetaDataImpl> impl_;
 };
 
+// \brief DataPageStats is a proxy around format::DataPageHeader and
```

Comment:
Why define this class here if it's only referenced in `column_reader.h`?

Reply:
Done.

Reply:
I think there is an API question of how we can consolidate page skipping for inlined statistics and those from the index.  To avoid churn we can always move this around later though i need be.